packages/neovim/package.json

```diff
@@ -58,50 +58,27 @@
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/preset-env": "^7.25.3",
     "@babel/preset-typescript": "^7.24.7",
-    "@types/jest": "^29.5.12",
-    "@types/node": "^16.18.113",
-    "babel-jest": "^29.7.0",
-    "jest": "^29.7.0",
-    "jest-haste-map": "^29.7.0",
-    "jest-resolve": "^29.6.1",
+    "@types/mocha": "^10.0.9",
+    "@types/node": "16.9.x",
+    "c8": "^10.1.2",
+    "expect": "^29.7.0",
+    "jest-mock": "^29.7.0",
```

Comment:
`jest-mock` is fine as a temporary step (if it saves some noise in this PR), but maybe we want to switch to https://sinonjs.org/ ? Or maybe it doesn't matter. (But maybe it's confusing to have both jest and mocha :)